ds-prep-python-tuples/main.py

```diff
@@ -0,0 +1,31 @@
+passenger = (12, True, "Bonnell, Miss. Elizabeth", "female", 58)
+print(passenger)
+name = passenger[2]
+print(name)
+age = passenger[-1]
+print(age)
+print(name, age)
```

Comment:
Please avoid going overboard with the `print` statements. It makes your output more difficult to examine:

---

![image](https://user-images.githubusercontent.com/16143771/149249531-7f679f24-7c78-45df-83f0-aba649c5abb6.png)

---

![image](https://user-images.githubusercontent.com/16143771/149249489-c83b0a68-33eb-47f1-889b-921cc307d378.png)